File: java-i/L1-013.java
//L1-013 计算阶乘和
/*
2021.11.2
21.37.08
By Xcl
*/
import java.util.Scanner;
public class Main {
	public static void main(String[] args) {
		Scanner s=new Scanner(System.in);
		int n=s.nextInt();
		int sum=0;
		for(int i=1;i<=n;i++) {
			int p=1;
			for(int j=1;j<=i;j++) {
				p=p*j;
			}
			sum=sum+p;
		}
		System.out.print(sum);
	}
}

File: c-c/L1-013.cpp
//L1-013 计算阶乘和
/*
2021.11.2
21.32.08
By Xcl
*/
#include<stdio.h>
int main()
{
	int n;
	int num=1,sum=0;
	scanf("%d",&n);
	for(int i=1;i<=n;i++)
	{
		num=1;
		for(int j=1;j<=i;j++)
			num*=j;
		sum+=num;
	}
	printf("%d",sum);
	return 0;
}
